tests/test_ci_skeleton.py

```diff
@@ -92,6 +92,11 @@
 *.pyc
 
 build_artifacts
+
+MacOSX11.0.sdk.tar.xz
```

Comment:
I wonder if we should use a glob here.

Reply:
For `11.0`?

Reply:
Yeah so we catch any sdk and don't have to come back and change it later.

Reply:
Makes sense. Regex?